### ISSUE TITLE
migrate-log.py reports "parsed losslessly" with no line for target-schema conformance, so a missing mandatory field is invisible until the first review

### ISSUE THREAD

**Issue by Panikkos88** [rebelytics/one-skill-to-rule-them-all#132]
**Scope first, because #119 covers the adjacent ground.** PR #119 already adds a `siblings_checked` placeholder to converted **resolved** entries, and deliberately leaves it absent on OPEN ones "so the review's sibling backfill still fires". This issue is not asking for that to change. It is about the migration **report**, which #119 does not touch, and which is what left me unaware the field was missing at all.

**What happened.** Converting a 9-entry pre-3.0 `log.md` with the bundled script, following `references/migration.md` exactly:

```
parsed 9 entries from 1 file(s)
flags: none
needs human review: 0/9
parsed losslessly:  9/9
```

Step 6's verification also passed — header count equals file count. Every signal said the migration was complete. Then:

```
$ ls observation-log/*.md | wc -l
9
$ grep -l '^siblings_checked:' observation-log/*.md | wc -l
0
```

All nine of my entries were OPEN, so #119's placeholder would not have applied to any of them. That is by design and I am not disputing it — the review did backfill the field the next day, exactly as intended. The problem is that nothing between the two moments said so. `flags: none` and `parsed losslessly: 9/9` are the only summary the adopter gets, and both are true statements about fidelity to the **source**, where the field did not exist. A field the source never had is precisely the field a source-fidelity check cannot report on.

**Why the gap is worth a line of output.** `siblings_checked` exists because a one-entry `skill:` list is byte-identical whether the siblings were evaluated or never considered — the field is the only thing making the absence of the judgement visible. Between migration and the first review, the whole backlog sits in exactly the indistinguishable state the field was invented to prevent, and nothing marks it. If the adopter's first review happens a week later, or the log is consulted before then, there is no signal. In my case the gap was real and load-bearing: all nine entries named one skill, and a family drift audit run the same day found that skill's shared material present in 1 of 8 members of its family.

**The contrast that gives it away.** The script *did* flag `status-missing` on 4 of 9 entries, and the overrides mechanism handled it cleanly. So it flags a field the legacy format defined and the writer omitted, and stays silent on a field the legacy format never defined at all. The louder failure is the safe one.

**Suggested fix — a report line, not a behaviour change.**

1. In `scripts/migrate-log.py`, print the two properties separately. Keep `parsed losslessly: N/M` for source fidelity, and add a target-conformance line naming each mandatory frontmatter field the converted set does not carry:

   ```
   target conformance: siblings_checked 0/9 — the review's sibling backfill will
                       populate these; nothing else will
   ```

   For entries #119 already covers, this line reads `9/9` and costs nothing.

2. In `references/migration.md` step 6, add the corresponding check so it is verifiable without reading the script's output twice:

   ```bash
   n=$(ls observation-log/*.md | wc -l)
   for field in status siblings_checked; do
     echo "$field: $(grep -l "^$field:" observation-log/*.md | wc -l)/$n"
   done
   ```

   Enumerate those fields from the target's own definition (SKILL.md → How to Log), not from the mapping table in `migration.md` — a field absent from the mapping table is indistinguishable from a field absent from the schema, which is the same blind spot one level up.

**Generalisable principle.** A migration has two success criteria that are routinely collapsed into one: did anything from the source get lost, and does the result satisfy the target's rules. Only the first is visible from the source. A conversion that reads the old format perfectly will report success while producing output that is uniformly non-conformant under the new one — and the fields this hits are precisely the ones the new schema added, which is to say the ones the new version exists for.

**Possibly related:** #107 ("the instrument-failure guards are enumerated per snippet, so each new instrument arrives unguarded"). This is arguably another instance — an instrument reporting on one of two possibilities and concealing the other. If #107's fix lands as a property of instruments rather than a per-snippet list, this may fall out of it.

---

*Reported from a real v3.1.0 migration. Deferring to #119 on anything that overlaps it.*

**Comment by rebelytics** [comment]
Thanks for this. In the queue and will make it into one of the next weekly releases.
